src/Illuminate/Foundation/Application.php

```diff
@@ -522,10 +531,7 @@ public function getProvider($provider)
 	{
 		$name = is_string($provider) ? $provider : get_class($provider);
 
-		return array_first($this->serviceProviders, function($key, $value) use ($name)
-		{
-			return $value instanceof $name;
-		});
+		return isset($this->registeredProviders[$name]) ? $this->registeredProviders[$name] : null;
```

Comment:
Well, this is just what @taylorotwell changed it to:
https://github.com/toddbc/laravel-framework/commit/bbff30f7b992bf096a76c3bdebfc82ede82a8017#diff-7b18a52eceff5eb716c1de268e98d55d

I would argue it should in any case have a `return null;`, for clarity.  But it doesn't matter, should I just change it to an `if` without a `return null;`?